main.py

```diff
@@ -1,104 +1,89 @@
 from Forms import LoginForm, SignUpForm
 from os import environ
 from werkzeug.utils import secure_filename
 from werkzeug.security import generate_password_hash, check_password_hash
 import boto3
 
+bp = Blueprint('myapp', __name__)
 
-app = Flask(__name__)
-app.config['SECRET_KEY'] = 'imageanalysissystem'
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:' + environ["DB_PASSWORD"] + '@lfiasdb.cwtorsyu3gx6.us-west-2.rds.amazonaws.com/postgres'
+loginManager = LoginManager()
 
+def create_app():
+    app = Flask(__name__)
 
-Bootstrap(app)
+    app.config['SECRET_KEY'] = 'imageanalysissystem'
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:' + environ["DB_PASSWORD"] + '@lfiasdb.cwtorsyu3gx6.us-west-2.rds.amazonaws.com/postgres'
 
-db = SQLAlchemy(app)
+    Bootstrap(app)
 
-loginManager = LoginManager()
-loginManager.init_app(app)
-loginManager.login_view = 'login'
-
-class Users(db.Model, UserMixin):
-    id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(50), nullable=False)
-    email = db.Column(db.String(50), unique=True, nullable=False)
-    password = db.Column(db.String(80), nullable=False)
-    events = db.relationship('Event', backref='users', lazy=True)
-    cameras = db.relationship('Camera', backref='users', lazy=True)
-
-class Event(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
-    camera_id = db.Column(db.Integer, db.ForeignKey('camera.id'), nullable=False)
-    type = db.Column(db.Enum(EventType), nullable=False)
-    timestamp = db.Column(db.DateTime, nullable=False)
-
-class Camera(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
-    name = db.Column(db.String(50), nullable=False)
-    status = db.Column(db.Enum(CameraStatus), nullable=False)
-    mode = db.Column(db.Enum(CameraMode), nullable=False)
-    
+    loginManager.init_app(app)
+    loginManager.login_view = 'myapp.login'
 
+    app.register_blueprint(bp)
 
+    from models.models import db
+    db.init_app(app)
 
+    return app
+    
 # EXAMPLE OF API ENDPOINT TIED TO SIMPLE-CLIENT
-
-@app.route("/pi", methods=["GET", "POST"])
+@bp.route("/pi", methods=["GET", "POST"])
 def index():
     notdata = 'test'
     if(request.method == "POST"):
         data = 'hello_world'
         return jsonify({'data': data})
-    return render_template("cameras.html")
-
+    return redirect(url_for('myapp.cameras'))
 # --------
 
-
 @loginManager.user_loader
 def loadUser(id):
     return Users.query.get(int(id))
 
-@app.route("/addEvent")
+@bp.route("/test", methods=["GET", "POST"])
+def test():
+    getData = 'get_request'
+    postData = 'post_request'
+    if(request.method == "POST"):
+        return jsonify({'data': postData})
+    elif (request.method == "GET"):
+        return jsonify({'data': getData})
```

Comment:
This is just a test route and is tested through some example tests as part of this PR. These are just placeholder, to show HOW to test JSON responses.